Sources/App/Controllers/API/API+PackageController+GetRoute+Model.swift

```diff
@@ -235,6 +235,17 @@ extension API.PackageController.GetRoute.Model {
             case executable
             case plugin
             
+            var stringValue: String {
+                switch self {
+                case .library:
+                    return "library"
+                case .executable:
+                    return "executable"
+                case .plugin:
+                    return "plugin"
+                }
+            }
+            
```

Comment:
```suggestion
```

We can remove this in favour of using `rawValue`.

Reply:
Hey, I suppose you added a conformation to `String` on the `ProductType`?